Modules/Shipping/Http/Services/ShippingService.php

```diff
@@ -53,15 +52,15 @@ public function createShipping(array $shippingData): Shipping
      */
     public function getShipping(): array
     {
-        $shippings = Shipping::where('user_id', auth()->user()->id)->get();
+        $shippings = User::find(auth()->user()->id)->getAllShippings()->leftJoin('countries', 'shippings.country', '=', 'countries.id')
+            ->select('shippings.*', 'countries.name AS shipping_country')->get();
```

Comment:
can we use this query instead of yours.
$shippings = User::with(['getAllShippings' => function ($query) {
        $query->leftJoin('countries', 'shippings.country', '=', 'countries.id')
            ->select('shippings.*', 'countries.name AS shipping_country');
    }])
    ->find(auth()->user()->id)
    ->getAllShippings;
    
    Get the relationships using eager loading